CHANGELOG.md

```diff
@@ -67,6 +67,7 @@
 - Minor: Migrated /uniquechat and /r9kbeta to Helix API. (#4057)
 - Minor: Migrated /uniquechatoff and /r9kbetaoff to Helix API. (#4057)
 - Minor: Make menus and placeholders display appropriate custom key combos. (#4045)
+- Minor: Make reply thread subtext easier to click (#4067)
```

Comment:
This is just part of the reply changelog, just add your changelog number to the big one at the top